### DISCUSSION TITLE
blinking status bar release apk with capacitor

### DISCUSSION THREAD

**Discussion by bobwatcherx** [quasarframework/quasar#15057] [CLI - Capacitor / Cordova modes]
![Screenshot_2022-12-05-16-38-33-59](https://user-images.githubusercontent.com/52072811/205604455-4013a6c4-4905-4860-ae4c-73375406e9e7.png)
i always have this kind of problem every build with capacitor . although other than with a quasar . but for the image above I made it with quasar . is there a solution for that

**Comment by metalsadman** [comment]
Where's the code?
